extensions/HyperV/hyperv.py

```diff
@@ -16,6 +16,9 @@
 # specific language governing permissions and limitations
 # under the License.
 
+import warnings
+warnings.filterwarnings('ignore')
```

Comment:
Running the status() in hyperv extension was causing this:
```
[root@qa1-421-xaas-demo-1ea23f9f-kvm-mgmt1 logs]# python3 /usr/share/cloudstack-management/extensions/HyperV/hyperv.py status /var/lib/cloudstack/management/extensions/HyperV/27b09915-2b95-4c87-bf17-9c37bca5cae1.json
/usr/local/lib/python3.6/site-packages/requests_ntlm/requests_ntlm.py:5: CryptographyDeprecationWarning: Python 3.6 is no longer supported by the Python core team. Therefore, support for it is deprecated in cryptography. The next release of cryptography will remove support for Python 3.6.
  from cryptography import x509
{"status": "success", "power_state": "poweroff"}
```

Reply:
The filter warning removes unnecessary string in stderr which is upsurped by mgmt server as extension output.